### PR TITLE
Add Elasticsearch logging integration.

### LINKED ISSUE #557: Add Elasticsearch logging integration.
Add Elasticsearch as a logging integration.

### PR DESCRIPTION
**Pull request by ggallen**
## Which Issue(s) this Pull Request Resolves

Resolves #557

## Release Note

```release-note
NONE
```

### CODE REVIEW

File: pkg/log_urls/log_urls_test.go [hunk 1]
```diff
@@ -23,17 +23,29 @@ func TestGenerateLogURLs(t *testing.T) {
...
...))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.container_name,negate:!f,params:(query:echo-3),type:phrase),query:(match_phrase:(kubernetes.container_name:echo-3)))),interval:auto,query:(language:kuery,query:''),sort:!())", "https://localhost:5601/app/discover#/?_a=(columns:!(),dataSource:(dataViewId:'035bffd1-b4b5-4123-af57-d89368cabfd8',type:dataView),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:_index,negate:!f,params:(query:fluentd),type:phrase),query:(match_phrase:(_index:fluentd))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.pod_id,negate:!f,params:(query:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'),type:phrase),query:(match_phrase:(kubernetes.pod_id:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.container_name,negate:!f,params:(query:echo-4),type:phrase),query:(match_phrase:(kubernetes.container_name:echo-4)))),interval:auto,query:(language:kuery,query:''),sort:!())"},
```
Comment (maruiz93) [inline pkg/log_urls/log_urls_test.go:47]: Any chance to have this line separated in several so that it's a little more readable? 
If it could be done I'd suggest to add the same approach for the rest of the `expected` parameters but this is specially hard to follow given the length.

**Review by maruiz93 — CHANGES_REQUESTED** [review]

File: integrations/logging/elasticsearch/install.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (ggallen) [reply integrations/logging/elasticsearch/install.sh:1]: Fixed.

File: integrations/logging/elasticsearch/uninstall.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (ggallen) [reply integrations/logging/elasticsearch/uninstall.sh:1]: Fixed.

File: pkg/log_urls/log_urls_test.go (hunk 1 above)
Comment (ggallen) [reply pkg/log_urls/log_urls_test.go]: Yeah, I hated this too.

I've moved the expected output to files, which I think should make things much better.

Comment (maruiz93) [reply pkg/log_urls/log_urls_test.go]: yes! I think it looks much better now :+1:

**Review by maruiz93 — APPROVED** [review]

**Review by rh-hemartin — APPROVED** [review]
